### PR TITLE
Update Command_overlord.java

### PR DESCRIPTION
**Pull request by ownre**
@DUFCLiam

### CODE REVIEW

**Comment by rovertdude** [comment]
Requesting for this to be tested.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_overlord.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             }
             catch (Exception ignored)
             {
-                playerMsg(ChatColor.WHITE + "Unknown command. Type \"help\" for help.");
+                playerMsg(ChatColor.WHITE + "You are not in the overlord list. Please contact an executive if this is in error.");
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_overlord.java]: Change this back, As this is a hidden command.

**Comment by rovertdude** [comment]
@DUFCLiam 
If they use the command, they obviously know about it.

Why hide it? There's no point.
Only I can let users use it.

**Comment by DUFCLiam** [comment]
But if someone accidentally does it in-game it's easy to mistake command... But when given power in a console you will review all your commands first.

**Comment by PuggyCrew** [comment]
Why do you need to close this?
Rovert never denied it; this is a valid PR.
@DUFCLiam

**Comment by ownre** [comment]
Is this fully tested quite yet?

**Comment by DUFCLiam** [comment]
No, you need to change one thing.

` playerMsg(ChatColor.WHITE + "You are not in the overlord list. Please contact an executive if this is in error.");`
back to
`playerMsg(ChatColor.WHITE + "Unknown command. Type \"help\" for help.");`

**Comment by ownre** [comment]
I do not have access to it.
What is wrong with it saying that?
It's not like they can do anything with it.

**Comment by DUFCLiam** [comment]
@ownre Please just change it back.

**Comment by ownre** [comment]
Fine.
Rovert will most likely change it back once.

**Comment by DUFCLiam** [comment]
Ready to merge!

**Comment by ownre** [comment]
"Tested" has a reason.
I'm fully obviously asking if the /ov command works.

**Comment by ownre** [comment]
Closing it will cause merging to be closed until opened; that's why rovert keeps them open.

**Comment by DUFCLiam** [comment]
I know I open them when its going to merge.

**Pull request merged**